1.data-exploration/scripts/nbconverted/2.test-train_split.py

```diff
@@ -1,37 +1,83 @@
 #!/usr/bin/env python
 # coding: utf-8
 
-# In[1]:
+# In[30]:
```

Comment:
Please ensure that the notebook is run sequentially prior to converting the notebook :D